test/integration/crd_rw_test.go

```diff
@@ -68,11 +67,11 @@ var _ = Describe(fmt.Sprintf("%s,%s: Basic CRD Read-Write", focusAws, focusAzure
 	)
 
 	AfterEach(func() {
-		result := CurrentGinkgoTestDescription()
-		if result.Failed {
+		result := CurrentSpecReport()
```

Comment:
This give a full spec report right, currently we are retrieving test level information (not spec level)
The migration guide says, to get test level information, we need to access the specific fields from the CurrentSpecReport().

https://github.com/onsi/ginkgo/blob/b453793b607ade42b5f4192ea53cd6a4244a73ff/docs/MIGRATING_TO_V2.md?plain=1#L485

Reply:
Yeah.